website/docusaurus.config.js

```diff
@@ -24,6 +25,23 @@ const config = {
   trailingSlash: false,
   markdown: {
     mermaid: true,
+    parseFrontMatter: async params => {
+      // Reuse the default parser
+      const result = await params.defaultParseFrontMatter(params);
+
+      if (
+        result.frontMatter.title &&
+        String(result.frontMatter.title).match(/Release/) &&
+        String(result.frontMatter.title).match(/\d+\.\d+/)
+      ) {
+        let versionMatch = String(result.frontMatter.title).match(/(\d+\.\d+)/) ?? [];
+        let version = versionMatch[1] ? versionMatch[1] : '';
+        if (version) {
+          parseNotes(params.filePath, version);
```

Comment:
maybe adapt/change the method name as with a method parse I'm expecting a result while here it's also writing a file.

Also parseNotes should be async (then we await)

Reply:
here, this code is not tested at all, so should we move more stuff to the ts file and just call from the .js file a function

```
  parseFrontMatter: async params => {
      myFunction(params);
  }
```

and then it's inside the function that you do all the stuff (and then you have unit tests)